graph-builder/src/graph.rs

```diff
@@ -23,12 +23,15 @@ use failure::{Error, Fallible};
 use futures::Future;
 use lazy_static;
 pub use parking_lot::RwLock;
-use prometheus::{self, Counter, IntGauge};
+use prometheus::{self, histogram_opts, Counter, Gauge, Histogram, IntGauge};
 use serde_json;
 use std::collections::{HashMap, HashSet};
 use std::sync::Arc;
 use std::thread;
 
+pub const UPSTREAM_SCRAPE_BUCKETS: &[f64; 11] =
```

Comment:
I remember to have asked for a docstring on this public constant but it must've been swallowed. Please add a docstring here with a description of the values and their use.

Reply:
Oh, right, I added one in #154 only. Fixed